### PR TITLE
[Havoc DH] remove nemesis and fel mastery

### PR DESCRIPTION
**Pull request by ToppleTheNun**
remove talents that were removed in either BfA or Shadowlands prepatch.

### CODE REVIEW

File: analysis/demonhunterhavoc/src/CHANGELOG.tsx [hunk 1]
```diff
@@ -4,6 +4,11 @@ import { Elodiel, flurreN, LeoZhekov, ToppleTheNun } from 'CONTRIBUTORS';
 import { SpellLink } from 'interface';
 
 export default [
+  change(
+    date(2022, 7, 24),
+    <>Remove talents that were removed in BFA/Shadowlands prepatch.</>,
+    ToppleTheNun,
+  ),
```
Comment (Pewtro) [inline analysis/demonhunterhavoc/src/CHANGELOG.tsx]:
```suggestion
  change(date(2022, 7, 24), 'Remove talents that were removed in Shadowlands.', ToppleTheNun),
```

**Review by Pewtro — CHANGES_REQUESTED** [review]
A minor thing - you don't have to create fragments (<> </>) when you're just writing text.

**Comment by ToppleTheNun** [comment]
@Pewtro This is ready for re-review whenever you're ready.

**Review by Pewtro — APPROVED** [review]
Looks good, thanks!

**Pull request merged**